### DISCUSSION TITLE
I deleted my organization, but my personal Github Copilot account still says it's managed by my organization.

### DISCUSSION THREAD

**Discussion by nor001** [community/community#140149] [Copilot Conversations]
### Select Topic Area

Question

### Body

When an organization is deleted, are all member assignments and GitHub Copilot removed? Or do I need to recover the organization and delete it manually?

**Comment by Akash1134** [comment]
Hi @norman003 

When an organization is deleted on GitHub, all member assignments and Copilot seats are automatically removed. For more information, you can refer to the [Canceling Copilot for your organization](https://docs.github.com/en/enterprise-cloud@latest/copilot/managing-copilot/managing-github-copilot-in-your-organization/managing-the-copilot-subscription-for-your-organization/canceling-copilot-for-your-organization) documentation. Regarding your personal account still being associated with the Org, before deletion did you ensured that all assigned Copilot seats were removed? 

Thanks!

**Reply by nor001** [reply]
Hi @Akash1134 

Before the deletion, the assigned Copilot positions were not deleted. What can I do in this situation? Should the organization be recovered and this step be taken?

Thanks!

**Reply by Akash1134** [reply]
The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
